Makefile

```
@@ -147,6 +147,9 @@ TEST_SYMBOLIC_BACKEND       := haskell
 
 tests/proofs/memory-concrete-type-spec.k%prove: PROVE_OPTIONS=--z3-tactic "(or-else (using-params smt :random-seed 1))" --z3-impl-timeout 5000
 tests/proofs/memory-concrete-type-spec.k%prove: KPROVE_MODULE=MEMORY-CONCRETE-TYPE-LEMMAS
+tests/proofs/wrc20-spec.k.prove: TEST_SYMBOLIC_BACKEND=java
+tests/proofs/wrc20-spec.k%prove: KPROVE_MODULE=WRC20-LEMMAS
+tests/proofs/locals-spec.k.prove: TEST_SYMBOLIC_BACKEND=java
```

Unless this proof has been modified, it should now pass on the Haskell backend:

```suggestion
tests/proofs/locals-spec.k.prove: TEST_SYMBOLIC_BACKEND=java
```

---

I'm happy to try it, let's see what if it passes.